source/crud/crud.rst

```diff
@@ -132,10 +132,11 @@ Read
     /**
      * Runs an aggregation framework pipeline.
      *
-     * Note: $out is a special pipeline stage that causes no results to be returned
-     * from the server. As such, the iterable here would never contain documents. Drivers
-     * MAY setup a cursor to be executed upon iteration against the $out collection such
-     * that if a user were to iterate a pipeline including $out, results would be returned.
+     * Note: $out and $merge are special pipeline stages that cause no results
+     * to be returned from the server. As such, the iterable here would never
+     * contain documents. Drivers MAY setup a cursor to be executed upon
+     * iteration against the output collection such that if a user were to
+     * iterate the return value, results would be returned.
```

Comment:
In [SPEC-1289](https://jira.mongodb.org/browse/SPEC-1289), I previously stated an intention to prohibit returning a cursor for `$merge` based on discussion in server scope documents. I later realized that could cause problems for strictly typed drivers that must return an iterable and decided against it.

Reply:
@jyemin is this what you and Charlie decided?

Alternatively, we could mandate that drivers must return an empty cursor from $merge.

Reply:
Per conversation in https://github.com/mongodb/specifications/pull/563#discussion_r291723210, let me know if we should add language advising drivers to apply `batchSize` to any cursor created directly on the output cursor (for `$out` and/or `$merge`).

I don't think we need to require drivers to avoid creating such an output cursor for `$merge` if they already do so for `$out`. AFAIK, such cursors are still only created in a pre-query state so a user could simply ignore them and avoid running any query on a potentially much larger output collection.

Reply:
@ShaneHarvey we didn't discuss this.  I agree with Jeremy that it would be weird to return an Iterable for $out and null for $merge, and it's best to just return an Iterable in all cases.   Applications are free to ignore this iterable

Reply:
Thanks for clearing that up. SGTM!

Reply:
Sounds like no action required here. Resolving.